src/status_im/ui/screens/chat/utils.cljs

```diff
@@ -3,6 +3,8 @@
             [status-im.utils.gfycat.core :as gfycat]
             [status-im.utils.platform :as platform]
             [status-im.i18n :as i18n]
+            [status-im.constants :as constants]
+            [taoensso.timbre :as log]
```

Comment:
Leftover?
```suggestion
```

Reply:
Yes, thanks for pointing this out, removed.